content/en/docs/contribute/_index.md

```diff
@@ -29,35 +29,35 @@ about contributing to Kubernetes.
 
 This website is maintained by [Kubernetes SIG Docs](/docs/contribute/#get-involved-with-sig-docs).
 
-Kubernetes documentation contributors:
+Kubernetes docs contributors can:
```

Comment:
This doesn't improve clarity.

Reply:
Do you mean next? 
```suggestion
Kubernetes documentation contributors can:
```

Reply:
Yes. Replacing documentation with docs improves neither readability nor accessibility IMO.